src/config.rs

```diff
@@ -1,10 +1,11 @@
-use crate::types::Rank;
```

Comment:
There doesn't seem to be much documentation on what the correct import order is supposed to be, but `rustfmt` does seem to want to put `crate` imports on top. I had a look at the official Rust repo, and they seem to be (mostly) going with:

1. `crate` imports
2. external imports
3. `std` imports

Perhaps we could go with that instead?

Reply:
Thanks for doing the legwork on researching the "standardized" (if only by popularity) order. I originally used what I do in all my projects, but as long as they're separated in some way, I don't mind which order it ends up being.